classes/task/insert_recordings.php

```diff
@@ -65,8 +65,10 @@ public function execute() {
                 $record->start_time = $rec->recording_start;
                 $record->end_time = $rec->recording_end;
                 $record->status = $rec->status;
-                $DB->insert_record('zoom_recordings', $record);
-                $DB->update_record('event', (object)['id' => $value->id, 'recording_created' => 1]);
+                $hh = $DB->insert_record('zoom_recordings', $record);
```

Comment:
@nayanalinkstreet I used this variable name to just show how it is done. Can you please change this to something meaningful?

Reply:
@Naveen1895 Changed the variable name